sample/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="black">#FF000000</color>
+    <color name="white">#FFFFFFFF</color>
+    <color name="caribbean_green">#00d59e</color>
+    <color name="green_blue">#01B183</color>
+    <color name="sky_blue">#5c9efa</color>
+    <color name="royal_blue">#3366FF</color>
+    <color name="energy_yellow">#F9D350</color>
+    <color name="alto">#D8D8D8</color>
```

Comment:
Could you change this names for others more useful related to where are being used?

Reply:
These Colors are only used for Demo, the color of state bar and circles are passed as aparameter on DataModelState.